api/envoy/api/v2/route/route_components.proto

```diff
@@ -34,7 +34,7 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // host header. This allows a single listener to service multiple top level domain path trees. Once
 // a virtual host is selected based on the domain, the routes are processed in order to see which
 // upstream cluster to route to or whether to perform a redirect.
-// [#next-free-field: 21]
+// [#next-free-field: 23]
```

Comment:
No more changes allowed to v2, it's frozen :)

Reply:
I understand that.
I reverted them and my build seg-faulted.  I can't get it to build *without* the changes.

Reply:
pastebin of 
1) copying v3 changes into a clean build
2) running fix_proto
3) showing the expected files are changed
4) building tests fails

https://pastebin.com/N8vSXHQM